de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/dialogs/MetadataTemplateViewDialog.java

```diff
@@ -100,45 +111,63 @@ public MetadataTemplateViewDialog(MetadataView.Presenter presenter,List<Avu> tem
 
                 avu.setValue(value);
             }
-
             avus.add(avu);
-        }
+
+        });
         return avus;
     }
-    
+
     public boolean isValid() {
-      List<IsField<?>> fields = FormPanelHelper.getFields(widget);
-            for (IsField<?> f : fields) {
-                if (!f.isValid(false)) {
-                    valid = false;
-                }
+        List<IsField<?>> fields = FormPanelHelper.getFields(widget);
+        for (IsField<?> f : fields) {
+            if (!f.isValid(false)) {
+                valid = false;
             }
+        }
 
-      return valid;
+        return valid;
     }
-    
+
     private void buildAvuMap() {
-    	for (MetadataTemplateAttribute attribute: attributes) {
-    		Avu md = findMetadataForAttribute(attribute.getName());
-    		templateAttrAvuMap.put(attribute.getName(),md);
-    	}
+        for (MetadataTemplateAttribute attribute : attributes) {
```

Comment:
Same for this method, it has enough logic that should probably go in the presenter and be tested.